src/App.vue

```diff
@@ -102,6 +102,9 @@ export default {
           if (this.session.isSpectator) return;
           this.$store.commit("toggleModal", "roles");
           break;
+        case "v":
+          this.$store.commit("toggleModal", "voteHistory");
```

Comment:
The voteHistory modal should only open when there is an actual voteHistory to show:
```suggestion
          if (this.session.voteHistory.length) {
            this.$store.commit("toggleModal", "voteHistory");
          }
```